### ISSUE TITLE
Unexpected ENOENT error when deploying StandaloneERC20

### ISSUE THREAD

**Issue by mverzilli** [OpenZeppelin/openzeppelin-sdk#1516]
### Local setup

OS: macOS Catalina 10.15.3
Node version: v12.16.1
npm version: 6.13.4

@openzeppelin/cli version: 2.8.0
@openzeppelin/contracts-ethereum-package: 2.4.0

### Reproducing the issue

Follow the instructions in https://docs.openzeppelin.com/cli/2.8/dependencies until you complete the [Creating an ERC20 Token](https://docs.openzeppelin.com/cli/2.8/dependencies#creating-an-erc20-token).

Expected: a console trace more or less similar to the one shown in the docs.
Actual: after the contract instance is created, an error is printed to the console:

```
✓ Instance created at 0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66
/[...]/token-exchange2/build/contracts/StandaloneERC20.json: ENOENT: no such file or directory, open '/[...]/token-exchange2/build/contracts/StandaloneERC20.json'
```

~In spite of that report, the contract is successfully created, which makes the overall behaviour kind of confusing but doesn't block the feature.~

Update: the issue seems to be a bit more serious than I at first realised.

Running `npx oz balance --erc20 0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66` returns the expected value, so the address points somewhere.

But, when you run `npx oz send-tx` you can't pick the `StandaloneERC20` instance you tried to create at the previous step. You only get to choose contracts defined directly in your project (provided of course you deployed them).

This seems to happen only in `cli@2.8`. It works as expected @2.7.

**Comment by frangio** [comment]
Same issue reported in https://github.com/OpenZeppelin/openzeppelin-sdk/issues/1519. There's some useful information there as well.

**Comment by frangio** [comment]
I think this was introduced in #1448. Prior to that, `updateTruffleDeployedInformation` had a conditional so it only ran when the contract was local. 

https://github.com/OpenZeppelin/openzeppelin-sdk/blob/e1f2d7e37d083ea6b31babcfaae247a76fc5472f/packages/cli/src/models/network/NetworkController.ts#L833-L834

**Comment by frangio** [comment]
Fixed by #1522 and released in 2.8.1.
